e2e/genesis.json

```diff
@@ -0,0 +1,25 @@
+{
```

Comment:
In order to run the tests, this file is required, but included in the `.gitignore`.